kube-deploy/roles/deploy-kube/tasks/centos.yml

```diff
@@ -24,19 +24,10 @@
     gpgkey: https://packages.cloud.google.com/yum/doc/yum-key.gpg
             https://packages.cloud.google.com/yum/doc/rpm-package-key.gpg
 
-- name: Add docker repository with gpg keys
-  yum_repository:
-    name: docker
-    description: Repository for docker
-    baseurl: https://yum.dockerproject.org/repo/main/centos/7/
-    enabled: yes
-    gpgcheck: yes
-    gpgkey: https://yum.dockerproject.org/gpg
-
 - name: install kubeadm and required prereqs
   yum: name={{ item }} update_cache=yes state=present state=latest
   with_items:
-  - docker-engine
+  - docker
```

Comment:
Have you tried these [similar] changes with Ubuntu as well? I'd like to correct both with a single PR when possible. I will create an issue on our side to track the long term solution (on the Docker side).

Also, one thing to consider @intlabs is to use the same repo for upstream, but pin our docker install to a previous version. In fact, I like this approach better than removing the upstream yum/apt repos entirely and switching over to a vendor repo (since this is a little more in line with Kubernetes documentation).